docs/articles/expensify-classic/account-settings/Close-Account.md

```diff
@@ -1,5 +1,122 @@
 ---
-title: Close Account
+title: Close-Account.md
 description: Close Account
----
-## Resource Coming Soon!
+--- 
+# Overview 
+
+Here is a walk through of how to close an Expensify account through the website or mobile app. 
+
+# How to close your account 
+On the Web:
+
+1. Go to **Settings** in the left hand menu. Click **Account**. 
+2. Click “Close Account” 
+3. Follow the prompts to verify your email or phone number.
+4. Your account will be closed, and all data will be deleted.*
+
+![Insert alt text for accessibility here](https://help.expensify.com/assets/images/image-name.png){:width="100%"}
```

Comment:
Can you clarify what you mean by alt text? Thanks!

Reply:
> alt text 

it's a description of the image in case the image doesn't load for the user.
so we need to replace `Insert alt text for accessibility here` with a small description of the image

Reply:
Done!